aws-rotate-keys/README.md

```diff
@@ -0,0 +1,90 @@
+# AWS CircleCI key rotate orb
+
+This orb can be used to rotate AWS access keys and update corresponding CircleCI environment variables.
+A common use case would be to set up a scheduled CircleCI job which rotates access keys for the used account.
+
+## Executors
+This orb defines a small 'default' executor for running the aws commands.
+Any container that has aws cli and curl can run the this orb.
+
+## Commands
+### rotate
+This is the only command available in this orb. It rotates the AWS access keys for the account specified as the `aws-username` parameter. In addition of rotating the keys, this command also updates the corresponding environment variables. In order to run this command, you need to make sure that the aws cli client is already authenticated.
+
+**Parameters**
+- `aws-username` - user name of the AWS account you want to rotate keys for
+- `github-org` - name of the Github organisation the repository belongs to
```

Comment:
I assume this is so you can rotate the keys in a different repo to the one using the orb.
CircleCI exposes the current org/repo as environment variables, which would be good defaults.
I doubt anyone would need to change the github-org, for example.

Reply:
That's a good point. I've removed those and I use `$CIRCLE_PROJECT_USERNAME` and `$CIRCLE_PROJECT_REPONAME` instead.